index.js

```diff
@@ -159,4 +159,53 @@ mung.headersAsync = function headersAsync (fn) {
     }
 }
 
+mung.write = function write (fn, options = {}) {
+    return function (req, res, next) {
+        const original = res.write;
+        const mungError = options.mungError;
+
+        res.write = function (chunk, encoding, callback) {
+            if (res.headersSent) {
```

Comment:
`res.write` can be called after the headers are sent.  Please remove the check.

Reply:
Hmm, that's true. Will update. 

On that note, since `res.write` can be called multiple times before `res.end`, I'm thinking I'll also remove the `204` status check (we can't assume `No content` just because one call to res.write returns `null`).